src/events/event/EventPage.js

```diff
@@ -53,6 +54,7 @@ const EventPage = ({
   pageContext: { startTime, endTime },
   pageContext,
 }) => {
+  console.log('EVENT', contentfulEvent)
```

Comment:
Perhaps we should enable the `no-console` eslint rule to prevent these being committed by accident?

Reply:
oops

Reply:
I've added it into ticket `WEBREF-27`.